src/main/scheduler/scheduler.c

```diff
@@ -494,3 +604,9 @@ uint16_t getAverageSystemLoadPercent(void)
 {
     return averageSystemLoadPercent;
 }
+
+float schedulerGetCycleTimeMultipler(void)
```

Comment:
MultiplIer?

Reply:
This returns a float that can be used when setting filter timebases to account for the gyro rate differing slightly from the expected (nominally 125us). Not used at present, but available if we want very precise filter timing.

For example, in rpm_filter.c, we could used this to correct `currentFilter->looptime` in the following:
```
       biquadFilterUpdate(
            template, frequency, currentFilter->looptime, currentFilter->q, FILTER_NOTCH, weight);
```

Reply:
Doh! Fixed.

Reply:
Seems to still be missing the 'i'. Also I note that this function isn't used at all, so it could potentially just be removed?

Reply:
Not sure why that would be. Looked OK to me. Just done a `git push -f` to be sure; see https://github.com/SteveCEvans/betaflight/blob/525fdebf21b3c637ff8f27699019e56f666acf01/src/main/scheduler/scheduler.c#L610